### PR TITLE
#372 | Support new syscalls from kernel version 5.7

### LINKED ISSUE #372: Add support for new system calls
Since kernel version 5.1, new system calls have been added to Linux (~20 for x86_64 architecture).
Add support for these syscalls

### PR DESCRIPTION
**Pull request by mtcherni95**
Support new syscalls from kernel version 5.7 - Resolves #372

### CODE REVIEW

File: tracee/consts.go [hunk 1]
```diff
@@ -836,4 +854,22 @@ var EventsIDToParams = map[int32][]external.ArgMeta{
...
+	Fsopen:                     {{Type: "const char*", Name: "fsname"}, {Type: "unsigned int", Name: "flags"}},
+	Fsconfig:                   {{Type: "int", Name: "fs_fd"}, {Type: "unsigned int", Name: "cmd"}, {Type: "const char*", Name: "key"}, {Type: "const void*", Name: "value"}, {Type: "int", Name: "aux"}},
+	Fsmount:                    {{Type: "int", Name: "fsfd"}, {Type: "unsigned int", Name: "flags"}, {Type: "unsigned int", Name: "ms_flags"}},
+	Fspick:                     {{Type: "int", Name: "dirfd"}, {Type: "const char*", Name: "pathname"}, {Type: "unsigned int", Name: "flags"}},
+	PidfdOpen:                  {{Type: "pid_t", Name: "pid"}, {Type: "unsigned int", Name: "flags"}},
+	Clone3:                     {{Type: "struct clone_args*", Name: "cl_args"}, {Type: "size_t", Name: "size"}},
+	CloseRange:                 {{Type: "unsigned int", Name: "first"}, {Type: "unsigned int", Name: "last"}},
+	Openat2:                    {{Type: "int", Name: "dirfd"}, {Type: "const char*", Name: "pathname"}, {Type: "int", Name: "flags"}, {Type: "mode_t", Name: "mode"}},
```
Comment (yanivagman) [inline tracee/consts.go:870]: This doesn't fit the man page:
https://man7.org/linux/man-pages/man2/openat2.2.html (which have "how" and "size" arguments)

File: tracee/consts.go [hunk 2]
```diff
@@ -836,4 +854,22 @@ var EventsIDToParams = map[int32][]external.ArgMeta{
...
+	IoUringSetup:               {{Type: "unsigned int", Name: "entries"}, {Type: "struct io_uring_params*", Name: "p"}},
+	IoUringEnter:               {{Type: "unsigned int", Name: "fd"}, {Type: "unsigned int", Name: "to_submit"}, {Type: "unsigned int", Name: "min_complete"}, {Type: "unsigned int", Name: "flags"}, {Type: "sigset_t*", Name: "sig"}},
+	IoUringRegister:            {{Type: "unsigned int", Name: "fd"}, {Type: "unsigned int", Name: "opcode"}, {Type: "void*", Name: "arg"}, {Type: "unsigned int", Name: "nr_args"}},
+	OpenTree:                   {{Type: "int", Name: "dfd"}, {Type: "const char*", Name: "filename"}, {Type: "unsigned int", Name: "flags"}},
+	MoveMount:                  {{Type: "int", Name: "from_dfd"}, {Type: "const char*", Name: "from_path"}, {Type: "int", Name: "to_dfd"}, {Type: "const char*", Name: "to_path"}, {Type: "unsigned int", Name: "flags"}},
+	Fsopen:                     {{Type: "const char*", Name: "fsname"}, {Type: "unsigned int", Name: "flags"}},
+	Fsconfig:                   {{Type: "int", Name: "fs_fd"}, {Type: "unsigned int", Name: "cmd"}, {Type: "const char*", Name: "key"}, {Type: "const void*", Name: "value"}, {Type: "int", Name: "aux"}},
```
Comment (yanivagman) [inline tracee/consts.go:864]: There's still no man page for this one, but are you sure the first arg is int?
Here it is int*:  https://lwn.net/Articles/802096/

**Review by yanivagman — CHANGES_REQUESTED** [review]
Thanks @mtcherni95 !
This will bring our syscalls table to be up to date with recent kernels

File: tracee/consts.go (hunk 2 above)
Comment (mtcherni95) [reply tracee/consts.go]: Here I see it as int: https://patchwork.kernel.org/project/linux-fsdevel/patch/153313723557.13253.9055982745313603422.stgit@warthog.procyon.org.uk/

**Review by yanivagman — APPROVED** [review]

**Pull request merged**